### ISSUE TITLE
GT2 Belt Loop 760mm or 752mm?

### ISSUE THREAD

**Issue by xobes** [jyjblrd/4-Axis-Polar-3D-Printer#6]
GT2 Belt Loop 760mm

Looks like a 752mm loop was used and the tensioner is half-way -- would 750 be too big and cause tensioning to the limit?

![image](https://github.com/user-attachments/assets/9145df97-5849-4f34-9265-1bb1b97d1c65)

**Comment by jyjblrd** [comment]
You can always edit the 3d files to make the belt tensioner longer, but both lengths should be alright

**Comment by Chuckles182** [comment]
I just uploaded a remixed Ender 3 belt tensioner to be able to use a 760 mm belt.
https://www.printables.com/model/1137828-4-axis-x-axis-tensioner
![X Axis Tensioner](https://github.com/user-attachments/assets/bf216cdb-25de-4e60-ade1-02fb7acc568c)

**Comment by TheOnlyMann** [comment]
that's awesome, though the original design seems to use 2 more idlers to match the belt width- so i wonder how did oyu get around it.
you also seem to have changed up some of the mounting design to get aroudn the original design, which also would nice to be shared/commented upon!

**Comment by jyjblrd** [comment]
Yeah @Chuckles182 your build looks interesting, could you share more details / pictures about it?

**Comment by Chuckles182** [comment]
Where would be the best place to upload more info on my changes? I’m not very familiar with how GitHub works.

**Comment by jyjblrd** [comment]
maybe make a discussion post, you should be able to add images and files
